modules/installing-the-distributed-workloads-components.adoc

```diff
@@ -1,20 +1,19 @@
 :_module-type: PROCEDURE
 
-[id="configuring-the-distributed-workloads-components_{context}"]
-= Configuring the distributed workloads components
+[id="installing-the-distributed-workloads-components_{context}"]
+= Installing the distributed workloads components
 
 [role='_abstract']
-To configure the distributed workloads feature for your data scientists to use in {productname-short}, you must enable several components.
+To use the distributed workloads feature in {productname-short}, you must install several components.
 
 .Prerequisites
 ifdef::upstream,self-managed[]
-* You have logged in to {openshift-platform} with the `cluster-admin` role.
+* You have logged in to {openshift-platform} with the `cluster-admin` role and you can access the data science cluster.
 endif::[]
 ifdef::cloud-service[]
-* You have logged in to OpenShift with the `cluster-admin` role.
+* You have logged in to OpenShift with the `cluster-admin` role and you can access the data science cluster.
 endif::[]
```

Comment:
This isn't wrong, but you could remove ifdefs and just use:
* You have logged in to {openshift-platform} with the `cluster-admin` role.
* You have logged in to {openshift-platform} with the `cluster-admin` role and you can access the data science cluster.

Reply:
I think we should make all of our references to OpenShift consistent, but in a separate PR :)